### PR TITLE
(fix): no custom output folder is created with run-worker

### LINKED ISSUE #2154: Output dir doesn't auto create when tests run by command 'codeceptjs run-workers 5'
#### What are you trying to achieve?
I want to see my tests passed after running like `codeceptjs run-workers 5` and save screenshot 
#### What do you get instead?
instead I get error ` ENOENT: no such file or directory, open '/Users/WebstormProjects/e2e/.codeceptjs-output/test.png'`
because output dir was not created

### Details

* CodeceptJS version: 2.3.6
* Operating System: macos
* puppeteer

```js
exports.config = {
    tests: './user-scenario/**/*.test.ts',
    output: '../../.codeceptjs-output',
    plugins: {
        customLocator: {
            enabled: true,
            attribute: 'data-test',
            showActual: true,
        },
    },
    helpers: {
        Puppeteer: {
            waitForTimeout: 3000,
            waitForAction: 200,
            show: !!process.env.VISUAL,
            chrome: {
                args: ['--no-sandbox', '--disable-setuid-sandbox'],
                defaultViewport: {
                    width: 1280,
                    height: 960,
                },
                executablePath: process.env.CHROMIUM_PATH,
            },
        },
    },
    include: {
        I: './settings/steps.js',
    },
    bootstrap: null,
    mocha: {},
    name: 'e2e',
    require: ['ts-node/e2e'],
};
```

### PR DESCRIPTION
**Pull request by PeterNgTr**
## Motivation/Description of the PR
- Resolves #2154 

## Type of change

- [ ] Breaking changes
- [ ] New functionality
- [x] Bug fix
- [ ] Documentation changes/updates
- [ ] Hot fix
- [ ] Markdown files fix - not related to source code

## Checklist:

- [x] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [x] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/command/run-workers.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4859543&bid=16121377)

File: lib/command/run-workers.js [hunk 1]
```diff
@@ -59,6 +60,16 @@ module.exports = function (workers, options) {
   output.print(`Running tests in ${output.styles.bold(workers)} workers...`);
   output.print();
 
+  let outputDir;
+
+  if (path.isAbsolute(config.output)) outputDir = config.output;
```
Comment (elukoyanov) [inline lib/command/run-workers.js]: I think, better to use curly brackets blocks.
A few added strings will improve readability without ant problems.

https://github.com/airbnb/javascript#blocks--braces

File: test/runner/run_workers_test.js [hunk 2]
```diff
@@ -111,4 +110,13 @@ describe('CodeceptJS Workers Runner', function () {
       done();
     });
   });
+
+  it('should create output folder with custom name', function (done) {
+    if (!semver.satisfies(process.version, '>=11.7.0')) this.skip('not for node version');
+    exec(`${codecept_run_glob('codecept.workers-custom-output-folder-name.conf.js')} 1 --grep "grep" --debug`, (err, stdout, stderr) => {
+      stdout.should.include('customOutput_'); // feature
```
Comment (elukoyanov) [inline test/runner/run_workers_test.js]: As said in #2154 issue, when folder does not creates, error appears:
`ENOENT: no such file or directory, open '/Users/WebstormProjects/e2e/.codeceptjs-output/test.png'`

And it looks like, stdout has folder name always. No matter successful test run or not.

Did you check, that without your changes in `run-workers.js` test fails?

Comment (elukoyanov) [reply test/runner/run_workers_test.js]: Oh you check, that there are no `err`. Then ok, sorry

Comment (elukoyanov) [reply test/runner/run_workers_test.js]: Nevertheless, test name says codecept `should create output folder`, but it checks for stdout only, and no folder existing

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- test/runner/run_workers_test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4859543&bid=16121377)

File: test/runner/run_workers_test.js (hunk 2 above)
Comment (PeterNgTr) [reply test/runner/run_workers_test.js]: update the test to verify if the folder is created.

**Comment by DavertMik** [comment]
:rofl: all 3 CIs failed for different reasons not related to this PR

**Pull request merged**